tools/test-js-with-node

```diff
@@ -87,7 +87,8 @@ EXEMPT_FILES = make_set(
         "web/src/emoji_picker.js",
         "web/src/emojisets.js",
         "web/src/favicon.ts",
-        "web/src/feedback_widget.js",
+        "web/src/feature_flags.ts",
```

Comment:
This file was removed because it was deleted; don't add it back.

Reply:
fixed it